doc/configuration.md

```diff
@@ -53,7 +53,7 @@ User should chose following parameters combination (`clusterNetwork`+`defaultNet
 
 Multus will find network for clusterNetwork/defaultNetworks as following sequences:
 
-1. CRD object for given network name
+1. CRD object for given network name, in 'default' namespace
```

Comment:
@s1061123 Can you please explain this the I cant understand the combination between  clusterNetwork and the default namespace.

Reply:
Do you mean that the difference between 'clusterNetwork' and 'defaultNetwork'?

clusterNetwork: this network is used for Kubernetes cluster. This network is also used by Kubernetes (e.g. Pod IP).
defaultNetwork: this network is automatically added to all Pods (other than kube-system pod).

Both value takes CRD network-attachment-definition as you mentioned. Previously we don't mention that which namespace the network-attachment-definition is belonging to, hence I explicitly added.

Is that clear?

Reply:
I will prefer if this that the default will be the same namespace where the multus daemonset was deployed.

for security reasons we don't know who have access to the default namespace. a regular use can change this crd and brake all the cluster.

the default namespace for multus daemonset now is the kube-system namespace, a namespace that control all the main components of the k8s cluster. I think is the best place to put the clusterNetwork CRD. (hard coded for now works for me)

Reply:
I agree that we may put defaultNetwork/clusterNetwork in 'kube-system'. So let me change doc/code.

BTW, multus-cni plugin itself is invoked by kubelet from conatiner host, as far as I know, we could not identify multus-daemonset's namepsace and multus-daemonset itself.

Reply:
We can this to the daemonset
```yaml
      env:
        - name: MY_POD_NAMESPACE
          valueFrom:
            fieldRef:
              fieldPath: metadata.namespace
```